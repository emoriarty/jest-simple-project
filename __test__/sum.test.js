import sum from '../src/sum';

describe('testing sum function', () => {
  // Checks the result of sum against a constant value
  test('1 + 2 is equal to 3', () => expect(sum(1, 2)).toBe(3));
  // Checks the result of sum against a stored output snapshout
  test('2 + 2 is equal to 4', () => expect(sum(2, 2)).toMatchSnapshot());
});
