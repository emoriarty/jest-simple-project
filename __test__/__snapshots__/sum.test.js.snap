// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing sum function 2 + 2 is equal to 4 1`] = `4`;
